src/odemis/acq/milling/fibsemos.py

```diff
@@ -38,15 +37,23 @@
     )
     from fibsem.structures import FibsemMillingSettings, Point
     from fibsem.utils import load_microscope_configuration
-    OPENFIBSEM_INSTALLED = True
-except ImportError:
-    logging.warning("OpenFIBSEM is not installed. Please check the installation.")
+    FIBSEMOS_INSTALLED = True
+except ImportError as e:
+    logging.warning(f"fibsemOS is not installed or not available: {e}")
+    FIBSEMOS_INSTALLED = False
+
+
+def is_fibsemos_available() -> bool:
+    """
+    Check if fibsemOS is available for use.
+    """
+
+    return FIBSEMOS_INSTALLED
```

Comment:
I don't think this function is really needed. It's fine to directly use `FIBSEMOS_INSTALLED`.